### ISSUE TITLE
refactor(ledger-connector-fabric): upgrade to compatible with Validator implementation

### ISSUE THREAD

**Issue by sfuji822** [hyperledger/cacti#302]
**Is your feature request related to a problem? Please describe.**
Validators should have a common feature set to be part of the Ledger Plugin. PR#278 did not meet all functionality required for Validator. Still, the CACTUS core developer agreed on passing through it instead of rejecting the PR by adding a new issue item to remind fixing the identified issue.
 
**Describe the solution you'd like**
Fulfill missing functionalities such as sending signed-transaction(s), and refactoring to aligned to the Validator API in the white paper.

**Comment by RafaelAPB** [comment]
Does this issue imply only adding things to the connector or removing it as well?
The Fabric connector is providing support for several business logic plugins - which could mean that those would need to be refactored as well

**Comment by petermetz** [comment]
@sfuji822 I think this has been achieved in the meantime, but I'm not a 100% sure. Could you please confirm if we can close it or if it's still work in progress?

cc: @takeutak @izuru0 @outSH

**Comment by outSH** [comment]
@petermetz Can be closed once https://github.com/hyperledger/cacti/pull/2645 is merged
